waves/docker-compose.yml

```diff
@@ -2,15 +2,33 @@ version: '3'
 services:
   # RPC daemons
   liquid:
-    image: vulpemventures/liquid:latest
+    image: blockstream/elementsd:0.18.1.9
```

Comment:
Can we move all the unrelated changes into the previous commit? (This commit msg says "Add UI test to CI")